src/Components/GenreSelector/GenreSelector.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+const title = 'Genre Selector';
+
+export class GenreSelector extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selected: parseInt(props.selectedGenre),
+            genreList: props.genreList
+        };
+    }
+
+    selectGenre(genre){
+        let state = {...genre};
+        state.selected = genre.id;
+        this.setState(state);
```

Comment:
Do we need to keep id and selected in the state?
![image](https://github.com/AlieksieievOU/react-netflix-roulette/assets/40394400/e3ace4f6-edd6-478d-8298-885d35a8d738)

as I see, it's the same value